test/integration/services/data/test-get-claims-by-status.js

```diff
@@ -23,8 +23,8 @@ describe('services/data/get-claims-by-status', function () {
       })
     })
 
-    it('should get 5 columns from 3 tables', function (done) {
-      claims.get('TEST')
+    it('should run get and return 1 row with 5 columns from 3 tables', function (done) {
```

Comment:
test case should describe the purpose of the check, e.g. `should return claim list array`